SQF/dayz_code/stringtable.xml

```diff
@@ -16858,6 +16858,119 @@
+		<Key ID="STR_EQUIP_NOTEBOOK_HERBS">
+			<English>Herbs Notebook</English>
+			<German>Kräuter Notizbuch</German>
+		</Key>
+		<Key ID="STR_EQUIP_NOTEBOOK_HERBS_DESC">
+			<English>That notebook contains many information about herbs. It is very well documented and illustrated with pictures.</English>
+			<German>Das Notizbuch beinhaltet viele Informationen über Kräuter. Es ist sehr gut dokumentiert und illustriert.</German>
+		</Key>	
+		<Key ID="STR_EQUIP_DOCUMENT_RANDOM">
+			<English>Random Document</English>
+			<German>Irgendein Dokument</German>
+		</Key>
+		<Key ID="STR_EQUIP_DOCUMENT_RANDOM_DESC">
+			<English>Looks like a normal document, nothing special.</English>
+			<German>Sieht wie ein normales Dokument aus, nicht besonderes.</German>
+		</Key>	
+		<Key ID="STR_EQUIP_DOCUMENT_MILITARY">
+			<English>Military Document</English>
+			<German>Militärdokument</German>
+		</Key>
+		<Key ID="STR_EQUIP_DOCUMENT_MILITARY_DESC">
+			<English>A military document with many information about weapons, ammunition, tanks and helicopters.</English>
+			<German>Ein Militärdokument mit vielen Informationen zu Waffen, Munition, Panzern und Helikoptern.</German>
+		</Key>
+		<Key ID="STR_EQUIP_DOCUMENT_TOPSECRET">
+			<English>Top Secret Document</English>
+			<German>Streng geheimes Dokument</German>
+		</Key>
+		<Key ID="STR_EQUIP_DOCUMENT_TOPSECRET_DESC">
+			<English>A document with top secret information but the inscription "Top Secret" is missing.</English>
+			<German>Ein Dokument mit streng geheimen Informationen, aber die Aufschrift "Streng Geheim" fehlt.</German>
+		</Key>	
+		<Key ID="STR_EQUIP_MANUAL_ELECTRONICS">
+			<English>Electronics Manual</English>
+			<German>Elektronik Handbuch</German>
+		</Key>
+		<Key ID="STR_EQUIP_MANUAL_ELECTRONICS_DESC">
+			<English>A manual about electronics and how to repair most common electronic items.</English>
+			<German>Ein Handbuch über Elektronik und wie man die am häufigsten vorkommenden elektronischen Gegenstände repariert.</German>
+		</Key>	
+		<Key ID="STR_EQUIP_MANUAL_CAMERA">
+			<English>CCTV Camera Manual</English>
+			<German>Überwachungskamera Handbuch</German>
+		</Key>
+		<Key ID="STR_EQUIP_MANUAL_CAMERA_DESC">
+			<English>A manual about how to build and the use of a CCTV Camera.</English>
+			<German>Ein Handbuch über das Bauen und die Nutzung einer Überwachungskamera.</German>
+		</Key>	
+		<Key ID="STR_EQUIP_MANUAL_REPAIR">
+			<English>Repair Manual</English>
+			<German>Reperaturhandbuch</German>
+		</Key>
+		<Key ID="STR_EQUIP_MANUAL_REPAIR_DESC">
+			<English>A repair manual about how to fix items.</English>
+			<German>Ein Reperaturhandbuch über das Reparieren von Gegenständen.</German>
+		</Key>		
+		<Key ID="STR_EQUIP_MAP_Military">
+			<English>Military Map</English>
+			<German>Militärkarte</German>
+		</Key>
+		<Key ID="STR_EQUIP_MAP_Military_DESC">
+			<English>A military map with many markers. It looks like that the map was used for a big military operation.</English>
```

Comment:
`It looks like that the map was used for a big military operation.`
should be
`It looks like the map was used for a big military operation.`